db/schema.rb

```diff
@@ -15,6 +15,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "pgcrypto"
```

Comment:
I cant seem to get rid of it. I manually removed the line and then I tried the following

rake db:drop
rake db:create
rake db:migrate (this one actual seems to insert that line into schema.rb for me)
rake db:reload
rake claims:demo_data

I did try run bundle install before all of it as well.